### ISSUE TITLE
Disable VCPKG using Premake5

### ISSUE THREAD

**Issue by ANF-Studios** [premake/premake-core#1579]
**What's your question?**
Is there an option to *disable* vcpkg?

**Anything else we should know?**
I basically want to set this to no:
![image](https://user-images.githubusercontent.com/68814933/105567566-cf0c3900-5d00-11eb-9160-d49e7e42a946.png)

**Comment by Gaztin** [comment]
The vcpkg support for Visual Studio seems to be a third party extension. Perhaps it has an options menu under "Tools" or "Extensions"?
If not, try setting it to "No", save the project, and take a look in the .vcxproj file to try and find the XML tag. That way you can do a simple override in premake to insert the tag yourself during the generator process.

**Comment by ANF-Studios** [comment]
Oh, whoops. I completely forgot about this issue, my bad. And, I did inspect the tag, but... how do I inject it? Do you have any link referencing the docs? I'm not the best with keywords.

**Comment by starkos** [comment]
Searching the current premake-core source code for "VCPKG" isn't turning up anything so I suspect there isn't any support for this yet. Always happy to accept contributions though!

Going to mark this closed as answered but feel free to reopen if you want to discuss.

**Comment by ANF-Studios** [comment]
Oh, that's unfortunate. Hey, if you accept additions for this IDE specific feature (as far as I know), maybe you can rename this issue and implement it in the future yourselves or perhaps someone else might.

**Comment by starkos** [comment]
You're welcome to submit a feature request. But everyone here is awfully busy trying to ship their own work; you're much more likely to see this in Premake if you tackle it yourself, since you have the need of it. The Visual Studio exporter the easiest one to modify. There's a [contribution guide here](https://github.com/premake/premake-core/blob/master/CONTRIBUTING.md#contributing-a-fix-or-feature), and feel free to ask questions, always happy to help new contributors!

**Comment by ANF-Studios** [comment]
Alright, thanks. But hey, just one more question, is there such a thing as *Premake extensions*? I think I can make a feature request for that, that'd be pretty cool if it isn't a thing.

**Comment by Gaztin** [comment]
Premake extensions are already a thing! You can read up on them [here](https://premake.github.io/docs/Developing-Modules) 😄

**Comment by ANF-Studios** [comment]
Uhhh, is there something wrong? All these links in bullets are broken and lead to a 404.

**Comment by starkos** [comment]
Whoa, not sure what's happening there, it works fine when the site is run locally. I'll look into it, in the meantime you can see the same content [on the wiki here](https://github.com/premake/premake-core/wiki/Developing-Modules).

**Comment by starkos** [comment]
Ok this is weird…I was in the process of writing up a bug report for it and it seems to have magically started working? Tried it multiple browsers too. Can anyone else confirm?

**Comment by ANF-Studios** [comment]
Nope, still the same for me. Tried on Chrome and Edge, both in incognito mode (to prevent cache).

**Comment by starkos** [comment]
@KyrietS Any idea why the links [here](https://premake.github.io/docs/Developing-Modules) don't work, while the links [here](https://premake.github.io/docs/Adding-New-Action) do? I'm rather stumped.

**Comment by KyrietS** [comment]
Oh sorry. I missed this thread somehow. 

I see that a link is pointing to `docs/Developing-Modules/adding-unit-tests` but it should go to `docs/Adding-Unit-Tests/`. I need to do some experiments locally. I will let you know when I fix it.

**Comment by KyrietS** [comment]
I've added a new issue #1634 for this. If I don't find a solution then I'll add an issue on Docusaurus repo.
